### PR TITLE
Add configuration examples for PodPresets (reopened)

### PR DESCRIPTION
**Pull request by eqrx**
**What this PR does / why we need it**:
Adds documentation for the PodPreset setting and fixes a bug

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cmd/config.go [hunk 1]
```diff
@@ -447,7 +452,10 @@ features:
   # default 'privileged' PodSecurityPolicy, plus RBAC rules to authorize
   # 'kube-system' namespace pods to 'use' it.
   podSecurityPolicy:
-    enable: {{ .EnablePodSecurityPolicy }}
+	enable: {{ .EnablePodSecurityPolicy }}
```
Comment (kron4eg) [inline pkg/cmd/config.go]: still tabs

File: pkg/cmd/config.go [hunk 2]
```diff
@@ -447,7 +452,10 @@ features:
...
+  # Enables PodPresets admission plugin in API server.
+  podPresets:
+	enable: {{ .EnablePodPresets }}
```
Comment (kron4eg) [inline pkg/cmd/config.go]: still tabs

File: pkg/features/podpresets.go [hunk 3]
```diff
@@ -43,12 +42,12 @@ func activatePodPresets(feature *kubeoneapi.PodPresets, args *kubeadmargs.Args)
...
-	currentConfig, hasConfig := args.APIServer.ExtraArgs[extraConfigFlag]
+	currentConfig, hasConfig := args.APIServer.ExtraArgs[runtimeConfigFlag]
 	var newConfig string
 	if hasConfig {
 		newConfig = currentConfig + "," + podPresetEnableValue
 	} else {
 		newConfig = podPresetEnableValue
 	}
-	args.APIServer.ExtraArgs[extraConfigFlag] = newConfig
+	args.APIServer.ExtraArgs[runtimeConfigFlag] = newConfig
```
Comment (xmudrii) [inline pkg/features/podpresets.go]: Don't we have a function in some other features that handle this appending automatically?

Comment (eqrx) [reply pkg/features/podpresets.go]: Found it

**Comment by xmudrii** [comment]
/retest
flake

**Pull request merged**